core/src/main/java/com/google/googlejavaformat/java/FormatFileCallable.java

```diff
@@ -232,7 +234,12 @@ private boolean writeOutput(Write write) {
         }
         return false;
       }
-      if (!new File(tempFileName).renameTo(new File(fileToFormat.fileName()))) {
+      try {
+        Files.move(
+            new File(tempFileName).toPath(),
+            new File(fileToFormat.fileName()).toPath(),
+            StandardCopyOption.REPLACE_EXISTING);
+      } catch(IOException e) {
         synchronized (outputLock) {
           errWriter.append(tempFileName).append(": cannot rename temp file").append('\n').flush();
```

Comment:
Maybe log the exception message, to be consistent with the error handling in this method?